test.md

```diff
@@ -0,0 +1 @@
+this file should be ignored by git
```

Comment:
- [ ] Please note that this file should be ignored by git , but it's still in your repo, easy way to fix this is to delete it manually and make sure you add it to .gitignore file , kindly fix that